src-java/testing/performance-tests/src/test/groovy/org/openkilda/performancetests/spec/EnduranceV2Spec.groovy

```diff
@@ -92,22 +100,17 @@ idle, mass manual reroute, isl break. Step repeats pre-defined number of times"
         }
 
         and: "Wait for blinking isls to get UP and flows to finish rerouting"
-        Wrappers.wait(WAIT_OFFSET + flows.size() * 0.5) {
+        Wrappers.wait(WAIT_OFFSET + flows.size() * 0.8) {
             def isls = northbound.getAllLinks()
             (topology.isls - brokenIsls).each {
                 assert islUtils.getIslInfo(isls, it).get().state == IslChangeType.DISCOVERED
             }
             assert northbound.getAllFlows().findAll { it.status == FlowState.IN_PROGRESS.toString() }.empty
         }
 
-        then: "All flows are writting stats"
+        then: "All Up flows are pingable"
         def allFlows = northbound.getAllFlows()
         def assertions = new SoftAssertions()
-        assertions.checkSucceeds {
-            statsHelper.verifyFlowsWriteStats(allFlows.findAll { it.status == FlowState.UP.toString() }*.id)
-        } ?: true
-
-        and: "All Up flows are pingable"
         def pingVerifications = new SoftAssertions()
         allFlows.findAll { it.status == FlowState.UP.toString() }.forEach { flow ->
             pingVerifications.checkSucceeds {
```

Comment:
How about checking that flow is pingable?

Reply:
Can be added, but doesn't really make profit since we always use virtual switches in this test, it is not designed to be run elsewhere